tests/network/vmi_networking.go

```diff
@@ -707,8 +713,7 @@ var _ = Describe(SIG("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:
 				networkData, err := cloudinit.NewNetworkData(
 					cloudinit.WithEthernet("eth0",
 						cloudinit.WithDHCP4Enabled(),
-						cloudinit.WithDHCP6Enabled(),
-						cloudinit.WithAddresses(""), // This is a workaround o make fedora client to configure local IPv6
+						cloudinit.WithDHCP6Disabled(),
```

Comment:
what about doing it Enabled

and then after you modify here
https://github.com/kubevirt/kubevirt/pull/14189/files#diff-6fdd74ae3ebff086dcf12ed84c781a5ae997ea4efae4763ec864e57b6afc1807R689
to add 

```
nmcli connection reload
nmcli connection up "System eth0"
```
(or part of those)

using the following flow manually worked (with both enabled on cloudinit)
```
[connection]
id=eth0
uuid=11111111-2222-3333-4444-555555555555
type=ethernet
interface-name=eth0
autoconnect=true

[ethernet]

[ipv4]
method=auto

[ipv6]
method=dhcp
dhcp-hostname=yourhost
never-default=true

chmod 600 /etc/NetworkManager/system-connections/eth0.nmconnection
nmcli connection reload
nmcli connection up "System eth0"
```